.github/ISSUE_TEMPLATE/kPackageRequest.md

```diff
@@ -0,0 +1,30 @@
+---
+name: New Package Request
+about: Request a package be created that does not currently exist in the Chocolatey Community Repository?
+---
+
+<!--
+* Please ensure the package does not already exist in the Chocolatey Community Repository - https://chocolatey.org/packages - by using a relevant search.
+
+* Please ensure there is no existing open package request.
+
+* Please ensure the issue title starts with 'RFP - ' - for example 'RFP - Adobe Reader'
+
+* Please ensure you have both the Software Project URL and the Software Download URL before continuing.
+
+NOTE: Keep in mind we have an etiquette regarding communication that we expect folks to observe when they are looking for support in the Chocolatey community - https://github.com/chocolatey/chocolatey-package-requests/blob/master/README.md#etiquette-regarding-communication
+
+PLEASE REMOVE THESE COMMENTS ONCE YOU HAVE READ THEM.
+
+-->
+## Checklist
+
+- [ ] The package I am requesting does not already exist on https://chocolatey.org/packages;
+- [ ] There is no open issue for this package;
+- [ ] The issue title starts 'RFP - ';
+- [ ] The download URL is public and not locked behind a paywall / login;
+
+## Package Details
+
+Software project URL :
+Direct download URL for the software / installer :
```

Comment:
Perhaps having a value for a summary of the software as well?

As well as having a section the user can input what he/she expects the package to do (outside of just installing the program)

Reply:
That's a good suggestion. Added.